drivers/SmartThings/bose/src/listener.lua

```diff
@@ -182,11 +181,9 @@ function Listener:try_reconnect()
       return
     end
     retries = retries + 1
-    log.info(string.format("Retry reconnect in %s seconds", RECONNECT_PERIOD))
+    log.info(string.format("Reconnect attempt %s in %s seconds", retries, RECONNECT_PERIOD))
     socket.sleep(RECONNECT_PERIOD)
```

Comment:
Possible TODO for a future release: use a backoff here scoped to the loop so it gets GC'd/reset on successful connection.